src/xunit.performance.api/XunitRunner.cs

```diff
@@ -4,108 +4,95 @@
 
 using System;
 using System.Threading;
-using System.Reflection;
 using Xunit.Runners;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
 
 namespace Microsoft.Xunit.Performance.Api
 {
-    public class XunitRunner
+    public static class XunitRunner
```

Comment:
Why does XunitRunner need to be static?  I can envision a world where multiple of these are created in a given process (sequentially).  It should be the case that nothing persists from the first one to the second one, and so doing this is likely to be easiest by having it be an instance.